src/handlers/shared/stop.ts

```diff
@@ -7,23 +7,17 @@ export async function stop(context: Context, issue: Context["payload"]["issue"],
 
   // is there an assignee?
   const assignees = issue.assignees ?? [];
-  if (assignees.length == 0) {
-    logger.error("No assignees found for issue", { issueNumber });
-    await addCommentToIssue(context, "````diff\n! You are not assigned to this task.\n````");
-    return { output: "No assignees found for this task" };
-  }
-
   // should unassign?
-  const shouldUnassign = assignees[0]?.login.toLowerCase() == sender.login.toLowerCase();
+  const userToUnassign = assignees.find((assignee) => assignee?.login.toLowerCase() === sender.login.toLowerCase());
 
-  if (!shouldUnassign) {
+  if (!whoToUnassign) {
     logger.error("You are not assigned to this task", { issueNumber, user: sender.login });
+    await addCommentToIssue(context, "```diff\n! You are not assigned to this task.\n```");
```

Comment:
Isn't the logger supposed to also generate the diff?

Reply:
I forgot about that @gentlementlegen 

[QA](https://github.com/ubq-testing/command-start-stop/issues/2#issuecomment-2245996914)